### DISCUSSION TITLE
How do you enable APIs

### DISCUSSION THREAD

**Discussion by Satishglk** [postalserver/postal#3132] [Help with using Postal]
New to postal.  Appreciate all the support. I have few questions.
1. Installed postal and was able to get SMTP to work.
2. How do we enable APIs. where do we get the API key? what would be the API urls? for example if my postal URL is "somedomain.com" what will the API url's be?

Thank you again!

**Comment by willpower232** [comment]
If you have got SMTP working then all you need to do is add another credential but change the type from SMTP to API and then you should be good to go.

The API is hosted alongside the web UI so in theory you would give your code https://postal.example.com or whatever you are using to open the web UI.
